cmd/ocm/create/cluster/cmd.go

```diff
@@ -595,6 +623,11 @@ func preRun(cmd *cobra.Command, argv []string) error {
 		return err
 	}
 
+	if args.interactive {
+		machineCIDR, serviceCIDR, podCIDR, hostPrefix := GetDefaultClusterFlavors(connection, args.flavour)
```

Comment:
is the order of variables/matching the return values? serviceCIDR and podCIDR seem inverted

Reply:
Fixed